gcloud/connection.py

```diff
@@ -15,6 +15,7 @@
 """Shared implementation of connections to API servers."""
 
 import json
+import threading
```

Comment:
1. Can you provide the error message so that I may share the learnings?
1. Can you put a newline between the stdlib packages and the non-stdlib packages? (I know it's not your original code, but let's make it better, blah blah blah)